src/components/ClientCard/ClientCard.jsx

```diff
@@ -20,24 +20,20 @@ const ClientCard = props => (
   </li>
 );
 
-const mapStateToProps = state => ({
-
-});
-
 
 const mapDispatchToProps = dispatch => ({
-  setActiveChat: chat => dispatch(setActiveChat(chat)),
+  setActiveChat: chatId => dispatch(setActiveChat(chatId)),
 });
 
 
 export default connect(
-  mapStateToProps,
+  () => ({}),
```

Comment:
How do you feel about that solution then? Honestly I'm trying to find the best possible middleground:

I havent tried this but would this work?
```
export default connect(
  null,
  mapDispatchToProps,
)(ClientCard);
```